commands/validators.js

```diff
@@ -0,0 +1,118 @@
+    const validatorsMap = new Map();
+    const nextValidatorsMap = new Map();
+    nextValidators.map((validator) => nextValidatorsMap[validator.account_id] = validator);
+    currentValidators.map((validator) => { 
+        if (!(validator.account_id in nextValidatorsMap)) {
+            result.removedValidators.push(validator);
+        }
+        validatorsMap[validator.account_id] = validator;
+    });
+    nextValidators.map((validator) => {
+        if (validator.account_id in validatorsMap) {
+            if (validatorsMap[validator.account_id].stake != validator.stake) {
+                result.changeValidators.push({new: validator, old: validatorsMap[validator.account_id]});
+            }
+        } else {
+            result.newValidators.push(validator);
+        }
+    });
+    return result;
+}
+
+module.exports = {
+    command: 'EXPERIMENTAL_validators',
+    desc: 'lookup validators and proposals',
+    handler: exitOnError(async (argv) => {
+        const near = await connect(argv);
+
+        const genesisConfig = await near.connection.provider.sendJsonRpc('EXPERIMENTAL_genesis_config', {});
+        const validators = await near.connection.provider.sendJsonRpc('validators', [null]);
+
+        // Calculate all required data.
+        const numSeats = genesisConfig.num_block_producer_seats + genesisConfig.avg_hidden_validator_seats_per_shard.reduce((a, b) => a + b);
+        const seatPrice = findSeatPrice(validators.current_validators, numSeats);
+        const nextSeatPrice = findSeatPrice(validators.next_validators, numSeats);
+
+        // Sort validators by their stake.
+        validators.current_validators = validators.current_validators.sort((a, b) => -new BN(a.stake).cmp(new BN(b.stake)));
+        validators.next_validators = validators.next_validators.sort((a, b) => -new BN(a.stake).cmp(new BN(b.stake)));
+
+        var validatorsTable = new AsciiTable();
+        validatorsTable.setHeading('Validator Id', 'Stake', '# seats', '% online', 'bls produced', 'bls expected');
+        console.log(`Validators (total: ${validators.current_validators.length}, seat price: ${utils.format.formatNearAmount(seatPrice, 0)}):`);
+        validators.current_validators.forEach((validator) => {
+            validatorsTable.addRow(
+                validator.account_id,
+                utils.format.formatNearAmount(validator.stake, 0),
+                new BN(validator.stake).divRound(seatPrice),
+                Math.floor(validator.num_produced_blocks / validator.num_expected_blocks * 100),
+                validator.num_produced_blocks,
+                validator.num_expected_blocks);
+        });
+        console.log(validatorsTable.toString());
+
+        if (validators.current_fishermen) {
+            console.log(`\nFishermen (total: ${validators.current_fishermen.length}):`);
+            var fishermenTable = new AsciiTable();
+            fishermenTable.setHeading('Fisherman Id', 'Stake');
```

Comment:
What do you think of adding a little text re what is a fisherman? or at least a link to docs.

Reply:
Seems like a weird place IMO. We don't have explanation for anything else, do we?
Like what is transaction, staking, validators is as unclear as fishermen.